events/ready.js

```diff
@@ -1,21 +1,4 @@
 module.exports = async client => {
-  // Why await here? Because the ready event isn't actually ready, sometimes
```

Comment:
Again, going back to where the application is going, rightfully it should be here so the settings can grab the application for the permission tree.

Reply:
fixed it